client/src/store/providers/apollo.tsx

```diff
@@ -21,19 +21,24 @@ const arrangerLink = createHttpLink({
   uri: `${ARRANGER_API}${PROJECT_ID}/graphql`,
 });
 
-const getAuthLink = (userToken: string) => setContext((_, { headers }) => ({
-  headers: {
-    ...headers,
-    authorization: userToken ? `Bearer ${userToken}` : '',
-  },
-}));
+const getAuthLink = () => (
+  setContext((_, { headers }) => (
```

Comment:
TBC I'm not very familiar with ApolloLink, for what I understand 'setContext' will be called every times a request is done ?

Reply:
ca va être appelé a chaque re-render du apolloClient (don't a chaque update de données)

regarde la section de caching: https://www.apollographql.com/docs/react/api/link/apollo-link-context/#caching-lookups